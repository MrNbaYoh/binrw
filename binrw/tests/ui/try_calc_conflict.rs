use binrw::BinRead;

#[derive(BinRead)]
struct Foo {
    #[br(try, calc(None))]
    a: Option<u8>,
}

fn main() {}
