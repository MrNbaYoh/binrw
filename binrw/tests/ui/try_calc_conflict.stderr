error: `try` is incompatible with `default` and `calc`
 --> $DIR/try_calc_conflict.rs:5:10
  |
5 |     #[br(try, calc(None))]
  |          ^^^
